create_book.rb

```diff
@@ -0,0 +1,19 @@
+require './library'
+
+class CreateBook
+  def initialize(library)
+    @library = library
+    @books = @library.books
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    book = Book.new(title, author)
+    books << book
+    puts ''
+    puts '#=>  Book created successfully'
+  end
```

Comment:
- [ ] As requested by the previous reviewer, Please fix the issue while creating a book, the app is throwing the below error it's because you should call the instance variable `@books` not `books`. On line 15 please use `@books` instead of `books` to fix this issue.

![image](https://github.com/aima98/OOP-School-Library1/assets/83803414/56b443b0-f98d-4af0-91e7-5eed994bf10d)